tor/core/user_interaction.py

```diff
@@ -328,3 +330,19 @@ def process_wrong_post_location(post):
             'Something went wrong with asking about a misplaced post; '
             'ignoring.'
         )
+
+def process_message(message: praw.models.Message, config):
+    author = message.author
+    username = author.name
```

Comment:
@TheLonelyGhost this will break if we receive a message without an author (e.g. anything from the Reddit platform itself).

Reply:
Crap. You're right. I'll address that ASAP